nano/node/bootstrap.cpp

```diff
@@ -2183,6 +2215,32 @@ void nano::bootstrap_server::receive_bulk_pull_account_action (boost::system::er
 	}
 }
 
+void nano::bootstrap_server::receive_frontier_req_action (boost::system::error_code const & ec, size_t size_a, nano::message_header const & header_a)
+{
+	if (!ec)
+	{
+		auto error (false);
+		nano::bufferstream stream (receive_buffer->data (), header_a.payload_length_bytes ());
```

Comment:
I think these should be size_a instead of payload_length_bytes () since that's the number of bytes actually in the buffer.

Reply:
Done